### PR TITLE
[ON HOLD] Make plugin cards smarter

### PR DESCRIPTION
**Pull request by remidej**
### What does it do?

- [X] Mark a plugin as "installed" on the marketplace page if it's already installed
- [x] Copy the install command using either Yarn or npm, depending on what the user uses

### Why is it needed?

For in app marketplace

### How to test it?

Go to /admin/marketplace route

### CODE REVIEW

File: packages/core/admin/server/controllers/admin.js [hunk 1]
```diff
@@ -45,11 +47,21 @@ module.exports = {
...
 
     return {
-      data: { currentEnvironment, autoReload, strapiVersion, nodeVersion, communityEdition },
+      data: {
+        currentEnvironment,
+        autoReload,
+        strapiVersion,
+        nodeVersion,
+        communityEdition,
+        dependencies,
```
Comment (alexandrebodin) [inline packages/core/admin/server/controllers/admin.js:62]: I'm not sure I understand why we would return the dependencies?

Comment (remidej) [reply packages/core/admin/server/controllers/admin.js:62]: That's a mistake it's a leftover from a previous approach, before I modified the core loader to get installed plugins. I'll remove it

**Comment by markkaylor** [comment]
Hey @remidej I'm not sure what happened but it looks like when you merged `master` this branch was out of sync with `strapi-market` and it had a bunch of commits and 259 files changed. 

I tried to update that branch but it created a lot of conflcts that I didn't understand. With some help from Hicham,  I created a new branch based on this one reverted back to before you merged master, create a new branch based off master and opened [this PR](https://github.com/strapi/strapi/pull/12634). 

I had tried before just changing the base branch here to another branch based on master but then changed it back  I guess that's why all the commits are gone now I guess....

**Comment by remidej** [comment]
It indeed looks like I created a mess. Thanks for looking into it. I think we should just reapply [the changes I made to mock API calls](https://github.com/strapi/strapi/pull/12610/commits/c3a22ad39b8d8ff979c733f237ad7f6735a612b2#diff-d7e08f7d5609628f82cfc9373f73063d7969ddbe652b10c6676b5549e3f4c1e7R4-R631) into your PR.

I'll close this PR in favor of the other one, we can always reopen it if needed.

**Pull request closed without merging**